app/Http/Requests/StoreVolumeMetadata.php

```diff
@@ -40,12 +40,12 @@ public function rules()
     {
         return [
             'metadata_csv' => [
-                'required_without_all:metadata_text,ifdo_file',
+                'bail:required_without_all:metadata_text,ifdo_file',
                 'file',
                 'mimetypes:text/plain,text/csv,application/csv',
                 new Utf8,
             ],
-            'metadata_text' => 'required_without_all:metadata_csv,ifdo_file',
+            'metadata_text' => 'bail:required_without_all:metadata_csv,ifdo_file',
```

Comment:
This field doesn't use `Utf8` so `bail` is not required.

```suggestion
            'metadata_text' => 'required_without_all:metadata_csv,ifdo_file',
```